sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/MessageLockToken.java

```diff
@@ -3,28 +3,28 @@
 
 package com.azure.messaging.servicebus;
 
-import java.util.UUID;
-
+import com.azure.messaging.servicebus.models.ReceiveMode;
 /**
- * Type representing the lock-token.
+ * Type representing the lock-token. The lock token is a reference to the lock that is being held by the Service Bus in
+ * {@link ReceiveMode#PEEK_LOCK} mode.
  */
 public interface MessageLockToken {
 
     /**
      * Gets the lock token.
      *
-     * @return {@link UUID} representing the lock-token.
+     * @return Lock token for the message.
      */
-    UUID getLockToken();
+    String getLockToken();
 
     /**
-     * Created the {@link MessageLockToken} given a {@link UUID}.
+     * Created the {@link MessageLockToken} given a {@link String}.
      *
-     * @param uuid to use for creating {@link MessageLockToken} instance;
+     * @param token {@link String} for creating {@link MessageLockToken} instance;
      *
      * @return The created {@link MessageLockToken} object.
      */
-    static MessageLockToken fromUuid(UUID uuid) {
-        return () -> uuid;
+    static MessageLockToken fromToken(String token) {
```

Comment:
`fromString` is a better fit. When you're doing this conversion, I'd check that this is a valid UUID and if not, throw an exception.